core/new-gui/src/app/workspace/service/undo-redo/undo-redo.service.ts

```diff
@@ -54,6 +57,8 @@ export class UndoRedoService {
       this.canUndoStream.next(this.canUndo());
 
       console.log("service can undo", this.canUndo());
+      const commandMessage: CommandMessage = { action: "deleteOperator", parameters: [""], type: "undo" };
```

Comment:
I understand this is trying to propagate the "undo" and "redo" event, but it doesn't seem right that the action is "deleteOperator".

Moreover, this implementation assumes all clients have the same undo and redo stack, which is not always true. For example, one newly joined client has empty undo/redo stack, and if it recieves an undo/redo command, it will fail. 
We should propagate the actual changes instead.

Reply:
Renamed the action. For the undo/redo stack problem, this PR probably won't address it, but will come up with a solution later.